src/components/settings/APIKeys.vue

```diff
@@ -1,53 +1,63 @@
 <template>
     <div>
-        <div class="add-btn">
-            <button class="btn btn-primary me-2" type="button" @click="$refs.apiKeyDialog.show()">
-                <font-awesome-icon icon="plus" /> {{ $t("Add API Key") }}
-            </button>
+        <div
+            v-if="settings.disableAuth"
+            class="mt-5 d-flex align-items-center justify-content-center my-3"
```

Comment:
The CI has not run on this branch.
I have no idea why, but I think addding another commit might fix it => lets test it

```suggestion
            class="mt-5 d-flex align-items-center justify-content-center my-3" 
```